app.py

```
@@ -168,78 +172,191 @@ def get_config():
-
-    levels = []
-    for level in levels_db:
-        level_dict = level.to_dict()
-        if level_dict.get("points_needed") == "negative_infinity":
-            level_dict["points_needed"] = -float('inf')
-        levels.append(level_dict)
-    return levels
+    """Cargar niveles de aura desde Supabase."""
+    try:
+        response = supabase.table('aura_level').select('*').order('level', desc=False).execute()
+        return response.data
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)